news/2020-08-26-new-featured-artist-kobaryo.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "New Featured Artist: Kobaryo"
+date: 2020-08-26 08:00:00 +0000
+---
+
+We're proud to welcome speedcore sensation **Kobaryo** aboard as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/96/header.jpg)
+
+A big name in the speedcore scene for many years, **Kobaryo** is a superbly talented Japanese producer who also leads the *HiTNEX TRAX* music label and is a member of the highly respected *HARDCORE TANO\*C* label, who are similarly renowned for their contribution to rhythm game music as a whole.
+
+His work has been incredibly popular all throughout the game since his first unofficial debut in osu! during late 2015, with a track appearing in the Finals pool for the osu!taiko World Cup 2016 and a [whole league of other maps](https://osu.ppy.sh/beatmapsets?sort=ranked_asc&q=kobaryo) in the years since, too many to list in one article alone.
```

Comment:
```suggestion
His work has been incredibly popular all throughout the game since his first unofficial debut in osu! during late 2015, with a track appearing in the Finals pool for the [osu!taiko World Cup 2016](/wiki/Tournaments/TWC/2016) and a [whole league of other maps](https://osu.ppy.sh/beatmapsets?sort=ranked_asc&q=kobaryo) in the years since, too many to list in one article alone.
```
thought adding a link to the tournament wiki page here would be nice

Reply:
good call!